tempColorTokenBuild.ts

```diff
@@ -0,0 +1,135 @@
+// @ts-nocheck
+import StyleDictionary from 'style-dictionary';
+import { copyFilesAndFolders } from './config/utilities/copyFilesAndFolders';
+import { w3cJsonParser } from './config/parsers/w3c-json-parser'
+import { platformCss } from './config/platforms/css';
+import { platformDocJson } from './config/platforms/docJson';
+import { platformScss } from './config/platforms/scss';
+import { platformJs } from './config/platforms/javascript';
+import { platformTs } from './config/platforms/typescript';
+import { platformTypeDefinitions } from './config/platforms/typesDefinitions';
+import { platformDeprecatedJson } from './config/platforms/deprecatedJson';
+import { colorToHexAlpha } from './config/tranformers/color-to-hex-alpha';
+import { colorToRgbAlpha } from "./config/tranformers/color-to-rgb-alpha";
+import { colorToHex6 } from "./config/tranformers/color-to-hex6";
+import { jsonDeprecated } from './config/tranformers/json-deprecated';
+import { scssMixinCssVariables } from './config/formats/scss-mixin-css-variables';
+import { javascriptCommonJs } from './config/formats/javascript-commonJs';
+import { javascriptExport } from './config/formats/javascript-export';
+import { typescriptExportDefinition } from './config/formats/typescript-export-defition';
+import { platformJson } from './config/platforms/json';
+import { namePathToDotNotation } from './config/tranformers/name-path-to-dot-notation';
+
+const buildPath = 'tokens-v2-private'
+const prefix = 'primer'
```

Comment:
can we make this a system argument or env var? We'd need to customise this for Brand.

Reply:
Yes, sure. Do you have a best practise recommendation for that?

Reply:
Not a big fan of this, but let's to it for now to get moving and see if we can improve this later.

Reply:
Currently our `color` tokens are not, this is why the prefix is set to `undefined`.

If we set it to `primer` it would be prefixed. However this is something we can define when we compared the outputs.
It is just on option.

Reply:
I think I just don't like that it is a giant function  and would prefer to provide configs via something like env or a file. 

But this is I guess personal taste. 😅